Exiled.Events/Patches/Events/Scp244/UpdateScp244.cs

```diff
@@ -0,0 +1,102 @@
+// -----------------------------------------------------------------------
+// <copyright file="UpdateScp244.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.Events.Patches.Events.Scp244
+{
+#pragma warning disable SA1313
+    using System;
+    using System.Collections.Generic;
+    using System.Diagnostics;
+    using System.Reflection.Emit;
+
+    using Exiled.API.Features;
+    using Exiled.Events.EventArgs;
+
+    using HarmonyLib;
+
+    using InventorySystem;
+    using InventorySystem.Items.Usables.Scp244;
+    using InventorySystem.Searching;
+
+    using Mirror;
+
+    using NorthwoodLib.Pools;
+
+    using UnityEngine;
+
+    using static HarmonyLib.AccessTools;
+
+    /// <summary>
+    /// Patches <see cref="Scp244DeployablePickup"/> to add missing event handler to the <see cref="Scp244DeployablePickup"/>.
+    /// </summary>
+    [HarmonyPatch(typeof(Scp244DeployablePickup), nameof(Scp244DeployablePickup.UpdateRange))]
+    internal static class UpdateScp244
+    {
+        private static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstruction> instructions, ILGenerator generator)
+        {
+            List<CodeInstruction> newInstructions = ListPool<CodeInstruction>.Shared.Rent(instructions);
+
+            Label normalProcessing = generator.DefineLabel();
+
+            LocalBuilder resultOfDotCheck = generator.DeclareLocal(typeof(int));
+
+            // Tested by Yamato and Undid-Iridium
```

Comment:
Don't really care, yknow, this is not Exiled Playground.

Reply:
This was for clarification beyond just git blame, to know who tested it, and who to yell at - I can remove it since it is exasperating to you.

Reply:
I mean, we can't really add comments for everything that has been tested and/or who tested it. You feel me?

Reply:
Sure, I understand - no worries.